roles/update/tasks/switchover.yml

```
@@ -1,19 +1,13 @@
 ---
 - name: Perform switchover of the leader for the Patroni cluster "{{ patroni_cluster_name }}"
   become: true
```

```suggestion
  become: true
```

become  is not required for this task.